eng/build-utils.ps1

```diff
@@ -333,7 +333,7 @@ function Make-BootstrapBuild([switch]$force32 = $false) {
 
   Run-MSBuild $projectPath "/restore /t:Pack /p:RoslynEnforceCodeStyle=false /p:RunAnalyzersDuringBuild=false /p:DotNetUseShippingVersions=true /p:InitialDefineConstants=BOOTSTRAP /p:PackageOutputPath=`"$dir`" /p:EnableNgenOptimization=false /p:PublishWindowsPdb=false $force32Flag" -logFileName "Bootstrap" -configuration $bootstrapConfiguration -runAnalyzers
   $packageFile = Get-ChildItem -Path $dir -Filter "$packageName.*.nupkg"
-  Unzip (Join-Path $dir $packageFile) $dir
+  Unzip (Join-Path $dir $packageFile.Name) $dir
```

Comment:
This change makes the script work on both Windows PowerShell and PowerShell Core.